.github/workflows/ci.yml

```diff
@@ -0,0 +1,15 @@
+name: CI
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install dependencies
+        run: |
+          apt-get update && apt-get install --yes --no-install-recommends make python3-pip
+          pip install black==23.7.0 flake8==6.0.0 isort==5.12.0
```

Comment:
Once we switch to `poetry`, we can create a dev dependency group for these - one of the benefits of `poetry`